pytest_fixtures/core/contenthosts.py

```diff
@@ -180,13 +180,21 @@ def container_contenthost(rhel7_contenthost, target_sat):
 
 @pytest.fixture
 def centos_host(request, version):
-    request.param = {"rhel_version": version.split('.')[0], "distro": "centos"}
-    with Broker(**host_conf(request), host_classes={'host': ContentHost}) as host:
+    request.param = {
+        "rhel_version": version.split('.')[0],
```

Comment:
Consider creating a version object here to use `.major`? Change from current behavior, def optional.